applications/FluidDynamicsApplication/python_scripts/cfl_output_process.py

```diff
@@ -0,0 +1,180 @@
+# Importing the Kratos Library
+import KratosMultiphysics
+import KratosMultiphysics.FluidDynamicsApplication as KratosCFD
+
+# other imports
+from KratosMultiphysics.time_based_ascii_file_writer_utility import TimeBasedAsciiFileWriterUtility
+
+from statistics import mean, stdev
+
+
+def Factory(settings, model):
+    if(type(settings) != KratosMultiphysics.Parameters):
+        raise Exception(
+            "expected input shall be a Parameters object, encapsulating a json string")
+
+    return CFLOutputProcess(model, settings["Parameters"])
+
+
+class CFLOutputProcess(KratosMultiphysics.Process):
+    """
+    Auxiliary base class to output total flow forces
```

Comment:
I think this description is not correct (copy & paste error?)

Reply:
Done